content/inside-rust/program-management-update-2025-08.md

```diff
@@ -0,0 +1,270 @@
+
+Their work largely happens publicly in [the Council's repository](https://github.com/rust-lang/leadership-council/issues).
+
+The representatives meet every other Friday and link these issues in their agenda. When they make a decision, they summarize the discussion and propose an FCP (final comment period) on the relevant issue. As with all FCPs in the Project, they're interested in any feedback people have until the comment period is closed. They review it all.
+
+If you want to see what the Council is up to, these issues are a great complement to the [Council meeting minutes](https://github.com/rust-lang/leadership-council/tree/main/minutes/sync-meeting) I'm taking and publishing on their behalf.
+
+[kangrejos]: https://kangrejos.com/
+
+To see this in practice, here is a proposal to send me to the annual [Rust for Linux workshop (Kangrejos)][kangrejos]:
+
+https://github.com/rust-lang/leadership-council/issues/217
+
+### Representative Selections
+
+Every six month, half of the Council's term ends.
+
+Infra, Lang, Libs and Mods teams are looking for a new representative and the nominations are now open.
+
+If you want to learn more or you're interested in representing your team, please read [Eric Huss's post announcing the selection](https://blog.rust-lang.org/inside-rust/2025/08/15/leadership-council-repr-selection/).
+
+
+## Rust Foundation Project Directors 2025
+
+This fall, [we're also looking for new Project Directors][pd-blog].
+
+[pd-blog]: https://blog.rust-lang.org/inside-rust/2025/08/20/electing-new-project-directors-2025/
+
+The Directors have staggered terms as well and half is up for election every year.
+
+This time it's Santiago Pastorino, Scott McMurray, and Jakob Degen's. None are seeking reelection.
+
+These are seats directly on the Rust Foundation board. The Project directors serve the interest of the Rust Project as a whole and sit alongside the Member Directors who represent companies funding the Foundation.
+
+Each resolution the Foundation passes must be approved by *both* Member and Project Directors separately. That means regardless of the size of the board, the Project has an equal voice in Foundation matters.
+
+You can nominate yourself or another person until 2025-09-18. Please [read the blog post for more information][pd-blog]. The Project is always looking for fresh faces and diverse voices.
+
+I am the [facilitator of the selection process](https://github.com/rust-lang/leadership-council/blob/main/policies/project-directorship/election-process.md#setup) this time around. That means I've authored the blog post above, proposed the timeline, and I'll seek out consent and statements from the nominees. I've also announced the election on zulip as well as an email that should reach all Project members and I'll see it all through, including facilitating the actual election process.
+
+## Bevy/gamedev followup
```

Comment:
@alice-i-cecile & @PROMETHIA-27: I've tried to add more context to the reflection & variadic generics fatures for folks who aren't as familiar with them.

Will you have time for a quick check to make sure this actually makes sense from Bevy perspective?